### DISCUSSION TITLE
Tooltip arrow problem in bootstrap 5 RTL

### DISCUSSION THREAD

**Discussion by saleh-esmaili** [twbs/bootstrap#35528] [Q&A]
Hello. I'm using bootstrap 5 RTL. I don't know is this a bug or I missed something, when i use tooltip, tooltip arrow direction is not toward my element! does bootstrap 5 RTL support tooltip?

**Comment by motahharm** [comment]
Hi @saleh-esmaili 

Can You send screen shot??
Do you use Bootstrap icons? chenge the icon.
